examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -52,24 +55,30 @@ const replaceOptions = {
 }
 
 const workboxOrInjectManifestEntry = {
-	// vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
+	// Vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
 	dontCacheBustURLsMatching: /-[a-f0-9]{8}\./,
-	globDirectory: './build/',
+	modifyURLPrefix: {
+		'client/': '/',
+		'prerendered/pages/': '/'
+	},
+	globDirectory: '.svelte-kit/output',
 	globPatterns: ['**/*.{js,css,html,ico,png,svg,webmanifest}'],
+	// TODO: change this when sequential fixed
+	// globPatterns: ['client/*.{js,css,ico,png,svg,webmanifest}', 'prerendered/pages/**/*.{html}'],
```

Comment:
this should be `// globPatterns: ['client/**/*.{js,css,ico,png,svg,webmanifest}', 'prerendered/pages/**/*.{html}'],`